### PR TITLE
Create table to store EU entry and exit dates

### PR DESCRIPTION
**Pull request by roshan-mathewtech**
[146](https://unep-wcmc.codebasehq.com/projects/cites-support-maintenance/tickets/146)

### CODE REVIEW

File: db/migrate/20221011142831_create_eu_country_statuses.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateEuCountryStatuses < ActiveRecord::Migration
```
Comment (lucacug) [inline db/migrate/20221011142831_create_eu_country_statuses.rb:1]: I'm not totally convinced by the name of this table, I would have gone with something like `eu_country_dates` maybe? let's have a chat about it

**Review by lucacug — CHANGES_REQUESTED** [review]

**Comment by roshan-mathewtech** [comment]
Changes are done. Thanks

File: app/models/eu_country_date.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class EuCountryDate < ActiveRecord::Base
+  attr_accessible :eu_accession_year, :eu_exit_year
+  belongs_to :geo_entity
+  validates :geo_entity, :presence => true
+
+  private
+  
+  def is_country
```
Comment (lucacug) [inline app/models/eu_country_date.rb:8]: this doesn't seem to be used anywhere here, we might need to add a `validate` line

**Review by lucacug — CHANGES_REQUESTED** [review]

File: app/models/eu_country_date.rb (hunk 2 above)
Comment (roshan-mathewtech) [reply app/models/eu_country_date.rb:9]: oops!! forgot to add it. Added it now
Thanks

**Comment by lucacug** [comment]
looks good now! Tagging as don't merge to use it as the feature branch of all this task

**Review by lucacug — APPROVED** [review]

**Comment by lucacug** [comment]
closing because superseded by this one #904

**Pull request closed without merging**